src/assets/custom/css/custom-softmod-feature.scss

```diff
@@ -0,0 +1,751 @@
+    border: 1px solid #fb7268!important;
+    text-align: center!important;
+    width: 94%!important;
+    display: block!important;
+    padding: 10px!important;
+    margin-bottom: 20px!important;
+}
+
+
+/********************************
+    Component - Lists
+********************************/
+
+.list-ticks {
+    list-style: none;
+    padding: 10px 0;
+    margin: 0;
+}
+
+.list-ticks li {
+    padding: 0 0 5px 30px;
+    position: relative;
+    font-weight: 700;
+}
+
+.list-ticks li:before {
+    content: '';
+    background-image: url('{{site.baseurl}}/assets/custom/img/softmod/icon-tick.svg');
+    background-size: 20px 20px;
+    background-repeat: no-repeat;
+    position: absolute;
+    border-radius: 10px;
+    width: 20px;
+    height: 20px;
+    background-color: transparent;
+    background-size: 20px;
+    left: 0px;
+    top: 2px;
+}
+
+/********************************
+    Component - Icon
+********************************/
+
+.icon-sm {
+    position: absolute;
+    width: 52px;
+    left: 0px;
+	stroke-dasharray: 1400;
+	stroke-dashoffset: 1400;
+    animation: dash 3s linear forwards;
+    fill: #FFFFFF;
+}
+
+.icon-sm circle {
+	opacity: 0;
+	animation: dot 0.5s linear forwards;
+	animation-delay: 2.2s;
+}
+
+.box-stroke {
+    fill: none;
+    stroke:#FFFFFF;
+    stroke-linecap:round;
+    stroke-linejoin:round;
+    stroke-width:4px;
+}
+
+@keyframes dash {
+    to {
+        stroke-dashoffset: 0;
+    }
+}
+
+@keyframes dot {
+	to {
+	    opacity: 1;
+	}
+}
+
+/********************************
+    Module - Hero
+********************************/
+
+.sm-hero {
+    background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-hero-dark.jpg');
+    background-size: cover;
+}
+
+.sm-hero__grid {
+    display: block;
+}
+
+.sm-hero__grid__grid-item--intro {
+	align-items: center;
+    display: flex;
+    z-index: 2;
+    position: relative;
+}
+
+.sm-hero__grid__grid-item--intro div {
+    margin: auto;
+    color: #FFFFFF!important;
+    padding-top: 50px;
+    padding-bottom: 50px;
+}
+
+.sm-hero__grid__grid-item--intro div p {
+    color: #FFFFFF!important;
+}
+
+.sm-hero__grid__grid-item--intro-image {
+    display: none;
+    overflow: hidden;
+}
+
+.sm-hero__grid__grid-item--intro-image img {
+    width: 110%;
+    height: auto;
+    bottom: 0;
+    left: 0;
+}
+
+.sm-hero__grid__grid-item--intro h1 {
+    padding-left: 80px;
+    line-height: 1;
+    padding-bottom: 10px;
+    font-size: 3.5;
+}
+
+@media (max-width: 991px) {
+    .sm-hero__grid__grid-item--intro h1 {
+        margin-top: 2rem;
+        margin-bottom: 2rem;
+    }
+    .sm-hero__grid__grid-item--intro p,
+    .sm-hero__grid__grid-item--intro a {
+        margin-left: 6rem;
+    }
+}
+
+@media (min-width: 992px) {
+
+    .sm-hero {
+        background-color: transparent;
+        background-image: none;
```

Comment:
These two rules are clearly an over-ride. Why not move the rules that are being overridden into the `max-width: 991px` breakpoint where they belong?